mitodl/bootcamp-ecommerce
# Fix for migration on homepage alumni section - #183

#### What are the relevant tickets?
[Ticket](https://trello.com/c/jquch13T/183-fix-migration-issues-on-homepage-alumni-section)

#### What's this PR do?
Modifies the `cms.0017` migration to avoid running into issues when applying this migration with existing `AlumniPage` instances in the CMS.

#### How should this be manually tested?
Run the migration once you have an `AlumniPage` child under a bootcamp run page. Make sure the migration runs without a hitch. You should be able to add another alumni section under the home page as well.
